packages/grpc-js-xds/scripts/xds_k8s_lb.sh

```diff
@@ -45,6 +45,11 @@ build_test_app_docker_images() {
     -t "${CLIENT_IMAGE_NAME}:${GIT_COMMIT}" \
     .
 
+  if [[ -n $KOKORO_JOB_NAME ]]; then
+    branch_name=$(echo "$KOKORO_JOB_NAME" | sed -E 's|^grpc/node/([^/]+)/.*|\1|')
+    tag_and_push_docker_image "${CLIENT_IMAGE_NAME}" "${GIT_COMMIT}" "${branch_name}"
+  fi
```

Comment:
New better way to do it:

```suggestion
  if is_version_branch "${TESTING_VERSION}"; then
    tag_and_push_docker_image "${CLIENT_IMAGE_NAME}" "${GIT_COMMIT}" "${TESTING_VERSION}"
  fi
```

Reply:
(introduced in https://github.com/grpc/grpc/pull/30027)

Reply:
I made a single change to update both parts of the code that would use `TESTING_VERSION`.